server/authentication/clientCredentials.ts

```diff
@@ -0,0 +1,9 @@
+import config from '../config'
+
+export default function generateOauthClientToken(
+  clientId: string = config.apis.hmppsAuth.apiClientId,
+  clientSecret: string = config.apis.hmppsAuth.apiClientSecret
+): string {
+  const token = Buffer.from(`${clientId}:${clientSecret}`).toString('base64')
```

Comment:
Is this needed? think superagent supports basic auth out the box - don't need to create own auth header